dali/operators/expressions/arithmetic.h

```diff
@@ -261,6 +276,19 @@ class ArithmeticGenericOp : public Operator<Backend> {
     // TODO(klecki): allocate memory for intermediate results and point the threads to them
   }
 
+  /**
+   * @brief Prepare lists of tiles for every task that we have to execute, filling the pointers
+   * to data.
+   */
+  void PrepareTilesForTasks(workspace_t<Backend> &ws) {
```

Comment:
I think it may be a free function returning tiles_per_task_ instead of setting it.

Reply:
Done